client/my-sites/checkout/cart/secondary-cart.jsx

```diff
@@ -78,16 +77,16 @@ class SecondaryCart extends Component {
 
 		if ( ! cart.hasLoadedFromServer ) {
 			return (
-				<Sidebar className={ cartClasses } hasSidebar={ this.props.hasSidebar }>
+				<ul className={ cartClasses }>
 					<CartMessages cart={ cart } selectedSite={ selectedSite } />
 					<CartSummaryBar additionalClasses="cart-header" />
 					<CartBodyLoadingPlaceholder />
-				</Sidebar>
+				</ul>
 			);
 		}
 
 		return (
-			<Sidebar className={ cartClasses } hasSidebar={ this.props.hasSidebar }>
+			<ul className={ cartClasses }>
```

Comment:
Just to explain the change here - I'm removing the `Sidebar` parent since after moving cart to the right, it no longer makes sense to call the cart a `Sidebar`.